stable/datadog/templates/daemonset.yaml

```diff
@@ -23,6 +23,11 @@ spec:
 {{ toYaml .Values.daemonset.podAnnotations | indent 8 }}
       {{- end }}
     spec:
+      {{- if .Values.datadog.seLinuxLabel }}
+      securityContext:
```

Comment:
So the only reason I did not do it that way is because i do not know if Datadog (the company) wants to support passing anything other than the container label change. So I would let them decide before modifying.

Reply:
It would be great actually. Ability for users to customize the `securityContext` is a useful feature.
Feel free to 
> copy the whole block in with toYaml